packages/core/src/layout/constants.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const slackChannel = {
+  name: '#backstage',
+  href: 'https://spotify.slack.com/archives/C54P1J36Z',
```

Comment:
Let's have something different here, perhaps if the url is not set we fall back to just the channel name? Or just make it `example.com` or smth